pkg/storage/doc.go

```diff
@@ -0,0 +1,11 @@
+// Package storage implements the storage layer of the application.
```

Comment:
what application? pls fix all such places 
also here i would suggest to not put details about application since you can reuse the storage not only for new-aggregator